packages/edit-post/src/editor.js

```diff
@@ -31,14 +31,28 @@ function Editor( {
 	initialEdits,
 	...props
 } ) {
+	// Get the rendering mode for the post type.
+	const postTypeRenderingMode = useSelect(
+		( select ) => {
+			const postTypeObject =
+				select( coreStore ).getPostType( initialPostType );
+			if ( postTypeObject && postTypeObject?.default_rendering_mode ) {
+				return postTypeObject.default_rendering_mode;
+			}
+
+			return 'post-only';
```

Comment:
@mcsf 
>Why the difference in signatures between the two hooks? I see one takes a WP_Post_Type instance, the other takes the type's name.

That's a good question, and I debated that myself when implementing. My initial thought there was that `"{$post_type}_default_rendering_mode"` would be targeted to a specific post type and receive that specific post type object. Where `post_type_default_rendering_mode` would be targeted to _all_ post types, and only the name would be needed to conditionally set a value.

I can see how that could be confusing though, so we can make them consistent. Do you have a preference on which would be better to receive? 

>Would amending the hook name from {$post_type->name}_default_rendering_mode to post_type_{$post_type->name}_default_rendering_mode help developers infer and trace it back to its origin in the future? Otherwise I worry that certain post type names could make things confusing or make it sound like it's a third-party hook (e.g. woo_product_default_rendering_mode)

A good point. Will definitely make this update.

Reply:
> and only the name would be needed to conditionally set a value.

Yeah, I tried to fill in the blanks and guessed that that could have been the motivation. :) 

> I can see how that could be confusing though, so we can make them consistent. Do you have a preference on which would be better to receive?

Honestly, this isn't a strong opinion. It's _barely_ an opinion. Consistency matters a little, but so do convenience, stability and economy. I suppose passing the `WP_Post_Type` instance would be my instinctual choice. But I'm OK with almost anything else, as long as it was deliberate!